tcms/testruns/tests/test_admin.py

```diff
@@ -91,3 +96,48 @@ def test_changelist_view_icon_column(self):
             """
             ),
         )
+
+    @parameterized.expand(
+        [
+            ("positive_tests", positive),
+            ("neural_tests", neutral),
+            ("negative_tests", negative),
+        ]
+    )
+    def test_test_execution_statuses_can_be_deleted_but_one_must_remain(
+        self, name, exe_statuses
+    ):
+        for exe_status in exe_statuses[1:]:
```

Comment:
FYI Now that `exe_statuses` is not a list anymore but of type QuerySet this is called slicing. It works similar to lists but on the DB level: https://docs.djangoproject.com/en/3.0/topics/db/queries/#limiting-querysets